WordPress/src/main/res/layout/media_picker_fragment.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:background="@android:color/white">
+
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/wp_stories_take_picture"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="end|bottom"
+        android:visibility="gone"
+        android:backgroundTint="@color/pink_50"
+        android:tint="@color/white"
```

Comment:
Comparing this with the `photo_picker_fragment.xml` it seems we have the following added:

```
    android:background="@android:color/white">                  <------

    <com.google.android.material.floatingactionbutton.FloatingActionButton
        android:id="@+id/wp_stories_take_picture"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_gravity="end|bottom"
        android:visibility="gone"
        android:backgroundTint="@color/pink_50"                     <------
        android:tint="@color/white"                     <------
```

not sure if it was removed from the `deprecated` class instead 🤔

Reply:
good point! I've implemented the changes from photo_picker_fragment here e48ff142f29e84adb0a2f63d1a782fd379ff8371